src/Umbraco.Web.UI.Client/src/packages/core/backend-api/types.gen.ts

```diff
@@ -16140,5 +16143,5 @@ export type GetWebhookLogsResponses = {
 export type GetWebhookLogsResponse = GetWebhookLogsResponses[keyof GetWebhookLogsResponses];
 
 export type ClientOptions = {
-    baseUrl: `${string}://${string}` | (string & {});
+    baseUrl: 'http://localhost:11000' | (string & {});
```

Comment:
Sanity check: Was this changed/committed on purpose?

Reply:
No, though it happened when I ran `npm run generate:server-api-dev`.  Related to the new fetch client perhaps?  I'll remove from this PR and let someone FE notice and decide if it's an expected change we want in this file.

Reply:
Thanks 😄